doctr/models/detection/core.py

```diff
@@ -158,7 +163,7 @@ def __call__(
             raise ValueError("incorrect input shape: all pages are expected to be multi-channel 2D images.")
 
         processed_batches = self.pre_processor(pages)
-        out = [self.model(batch, return_boxes=True, **kwargs)['boxes'] for batch in processed_batches]
-        out = [boxes for batch in out for boxes in batch]
+        out = [self.model(batch, return_boxes=True, **kwargs)['boxes + angles'] for batch in processed_batches]
```

Comment:
let's use "preds" as key

Reply:
I think this might be better to do this in a single line
```python
out = [(boxes, angles) for batch in processed_batches for boxes, angles in zip(self.model(batch, return_boxes=True, **kwargs)['preds'])]
````